packages/api/src/tasks/checkAlerts.ts

```diff
@@ -262,21 +262,16 @@ export const handleSendGenericWebhook = async (
     'Content-Type': 'application/json', // default, will be overwritten if user has set otherwise
     ...(webhook.headers?.toJSON() ?? {}),
   };
-
-  // BODY
-
-  let body = '';
-  if (webhook.body) {
-    const handlebars = Handlebars.create();
-    body = handlebars.compile(webhook.body, {
-      noEscape: true,
-    })({
-      body: escapeJsonString(message.body),
-      link: escapeJsonString(message.hdxLink),
-      title: escapeJsonString(message.title),
-    });
-  }
-
+  // BODY, for v2, currently don't support custom template, use {{title}} | {{body}} | {{link}} as default template
+  let body = '{{title}} | {{body}} | {{link}}';
```

Comment:
Can't we set this as a default value when creating a generic webhook?  Giving it another thought, I think we should just add a 'body' field in the UI. It's kind of critical imo to empower the generic webhook.

v1 design looks like
<img width="701" alt="Screenshot 2025-02-18 at 9 46 16 PM" src="https://github.com/user-attachments/assets/b3ec6e94-d628-4c3d-983a-9be22984984f" />

Reply:
yeah, I am going to set `{{title}} | {{body}} | {{link}}` as default if there is no webhook body mention.

Reply:
Also I am looking into the v2 UI part.

Reply:
add webhook body UI in V2
<img width="771" alt="image" src="https://github.com/user-attachments/assets/81b7f6f8-7155-42b9-8a9c-f6745a11381e" />